repos/spack_repo/builtin/packages/acts/package.py

```diff
@@ -423,18 +435,24 @@ class Acts(CMakePackage, CudaPackage):
     depends_on("python", when="+python")
     depends_on("python@3.8:", when="+python @19.11:19")
     depends_on("python@3.8:", when="+python @21:")
+    depends_on("py-numpy @2.2", when="@44:")
     depends_on("py-onnxruntime@:1.12", when="+onnx @:23.2")
     depends_on("py-onnxruntime@1.12:", when="+onnx @23.3:")
+    depends_on("py-particle @0.24", when="@44:")
     depends_on("py-pybind11 @2.6.2:", when="+python @18:")
     depends_on("py-pybind11 @2.13.1:", when="+python @36:")
     depends_on("py-pytest", when="+python +unit_tests")
+    depends_on("py-setuptools", when="@44:")
+    depends_on("py-sympy @1.13", when="@44:")
```

Comment:
```suggestion
    depends_on("py-sympy @1.13", when="@44:")
    # Fix the version when the next after 44.1.0 is released
    depends_on("py-hatchling", when="@44.1.1")
```
See https://github.com/acts-project/acts/pull/4770#issuecomment-3489785399, this will fix `main` builds.

Reply:
Thanks, updated.